### ISSUE TITLE
API request failed (400 HTTP error): Error about patch_set in commit "Ensure this field has no more than 255 characters."

### ISSUE THREAD

**Issue by rlembo06** [getsentry/sentry#36409]
### Environment

- Ubuntu 20.04.4 LTS
- Sentry SaaS (sentry.io)
- @sentry/cli@1.74.4
- @sentry/webpack-plugin: 1.18.8

### Steps to Reproduce

1. Renamed lot a files with long path (exceed 255 chars)
2. Commit it
3. Run command `sentry-cli releases set-commits repo@5.140.1 --auto --ignore-missing --ignore-empty`

### Expected Result

New Sentry release deployed

### Actual Result

```
error: API request failed

caused by: sentry reported an error: request failure (http status: 400)

Object({"commits": Object({"patch_set": Object({"path": Array([String("Ensure this field has no more than 255 characters.")])})})})

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.

Please attach the full debug log to all bug reports.
```

**Comment by kamilogorek** [comment]
This is limitation of the Sentry itself, so transferring the issue to this repo.

**Comment by barkbarkimashark** [comment]
@rlembo06 we pushed a change to increase the filename limit. Do you mind testing again and seeing if this is still happening?

**Comment by rlembo06** [comment]
> @rlembo06 we pushed a change to increase the filename limit. Do you mind testing again and seeing if this is still happening?

Unfortunately, the bug is still present.
Despite update of the latest version of `@sentry/webpack-plugin `(1.18.9) then `@sentry/cli` (2.3.1).

**Comment by barkbarkimashark** [comment]
@rlembo06 thanks for checking. Took a quick look and it looks like we need didn't complete the changes needed. I've linked a PR that should allow users to have increased commit file paths. When this is merged, could you test again and verify?

**Comment by rlembo06** [comment]
> @rlembo06 thanks for checking. Took a quick look and it looks like we need didn't complete the changes needed. I've linked a PR that should allow users to have increased commit file paths. When this is merged, could you test again and verify?

Yes of course! Do not hesitate to come back to me as soon as it is merged

**Comment by rlembo06** [comment]
@barkbarkimashark Hi, I retried the command, but the error is still present

**Comment by rlembo06** [comment]
@barkbarkimashark Grea t! it's work now ! thanks a lot !
